youtube_dl/extractor/americastestkitchen.py

```diff
@@ -60,7 +80,11 @@ def _real_extract(self, url):
             'url': 'https://player.zype.com/embed/%s.js?api_key=jZ9GUhRmxcPvX7M3SlfejB6Hle9jyHTdk2jVxG7wOHPLODgncEKVdPYBhuz9iWXQ' % video['zypeId'],
             'ie_key': 'Zype',
             'description': clean_html(video.get('description')),
+            'timestamp': parse_iso8601(video.get('publishDate')),
             'release_date': unified_strdate(video.get('publishDate')),
+            'upload_date': unified_strdate(video.get('publishDate')),
+            'episode_number': episode.get('number'),
+            'season_number': episode.get('season'),
```

Comment:
use `int_or_none`.

Reply:
same for `episode_number`(both lines where highlighted).

Reply:
My mistake. Just fixed this!